src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -610,19 +671,25 @@ private static function fetchContextURLForItem($item)
 		} elseif (DBA::isResult($conversation) && !empty($conversation['conversation-uri'])) {
 			$context_uri = $conversation['conversation-uri'];
 		} else {
-			$context_uri = str_replace('/objects/', '/context/', $item['parent-uri']);
+			$context_uri = $item['parent-uri'] . '#context';
 		}
 		return $context_uri;
 	}
 
+	private static function fetchSensitive($item_id)
```

Comment:
Please move this method to `Model\Item::isSensitive`.

Reply:
Done